### PR TITLE
Backport PR #16671: Fix some readme bits

### PR DESCRIPTION
**Pull request by story645**
trying to fix backport errors in #16671

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,20 +23,24 @@
 .. _Gitter: https://gitter.im/matplotlib/matplotlib
 
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-.. _NUMFocus: http://www.numfocus.org
+.. _NUMFocus: https://www.numfocus.org
 
+<<<<<<< HEAD
```
Comment (timhoffm) [inline README.rst]: There‘s still diff information in your commit.

Comment (story645) [reply README.rst]: 🤦‍♀️that wasn't how it looked on my local machine

File: README.rst [hunk 2]
```diff
@@ -23,20 +23,16 @@
 .. _Gitter: https://gitter.im/matplotlib/matplotlib
 
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-.. _NUMFocus: http://www.numfocus.org
+.. _NUMFocus: https://www.numfocus.org
 
-
-.. |GitTutorial| image:: https://img.shields.io/badge/PR-Welcome-%23FF8300.svg?
-.. _GitTutorial: https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project
```
Comment (QuLogic) [inline README.rst]: I don't think you want to remove this, because it's still there on line 1.

Comment (story645) [reply README.rst]: 🤦‍♀️ clearly I shouldn't do backports

**Pull request merged**